<?xml version="1.0" encoding="UTF-8"?>

<launch>

    <arg name="simulated_camera" default="true" />
    <arg name="camera_pan_angle" default="-0.605" />
    <arg name="mode" default="get_data" />
    <arg name="device_to_use" default="cpu" />
    <arg name="csi_camera" default="true" />
    <arg name="plot_images" default="true" />



    <node   name="collision_avoidance_training_node"
            pkg="my_ignisbot_collision_avoidance_pkg"
            type="collision_avoidance_training.py"
            respawn="false"
            output="screen"
            args="$(arg simulated_camera) $(arg camera_pan_angle) $(arg mode) $(arg device_to_use) $(arg csi_camera) $(arg plot_images)">
    </node>

    <node   name="simplegui_node"
            pkg="my_ignisbot_collision_avoidance_pkg"
            type="simplegui.py"
            respawn="false"
            output="screen"
            args="">
    </node>

</launch>